lib/agenda/job-processing-queue.ts

```diff
@@ -47,8 +47,8 @@ JobProcessingQueue.prototype.push = function(this: JobProcessingQueue, job: Job)
  */
 JobProcessingQueue.prototype.insert = function(this: JobProcessingQueue, job: Job) {
   const matchIndex = this._queue.findIndex(element => {
-    if (element.attrs.nextRunAt.getTime() <= job.attrs.nextRunAt.getTime()) {
-      if (element.attrs.nextRunAt.getTime() === job.attrs.nextRunAt.getTime()) {
+    if (element.attrs.nextRunAt!.getTime() <= job.attrs.nextRunAt!.getTime()) {
```

Comment:
nextRunAt is actually not always set, at least if i remember correctly. (never happened to me in production though), so good to go for me, but if we want to be "100%" safe, we should add an additional check like:
`if (element.attrs.nextRunAt && element.attrs.nextRunAt.getTime() <= job.attrs.nextRunAt!.getTime()) {`